modules/@ergonode/workflow/src/models/workflowDesigner/__tests__/WorkflowDesignerCalculations.test.js

```diff
@@ -142,3 +143,25 @@ describe('getMappedTransitions', () => {
         expect(result).toEqual(transitionsToSend);
     });
 });
+
+describe('getMappedStatusPositions', () => {
+    it('Based on layout element get statuses to send - empty', () => {
+        const result = getMappedStatusPositions({
+            layoutElements: [],
+            statuses,
+        });
+
+        expect(result.length).toEqual(0);
+        expect(result).toEqual([]);
+    });
+
+    it('Based on layout element get statuses to send', () => {
+        const result = getMappedStatusPositions({
+            layoutElements,
+            statuses,
+        });
+
+        expect(result.length).toEqual(3);
+        expect(result).toEqual(layoutElements);
```

Comment:
Not sure about this test - layoutElements are argument of the getMappedStatusPositions. If the result of it is equal the one of the argument then what is the purpose of that function?